Automattic/wordpress-activitypub
# Remove html_entity_decode to prevent encoding issues

Fixes #445

## Proposed changes:

Previously, content like `&lt;` was incorrectly converted to `<` - this meant that HTML elements would be passed which wouldn't render correctly.

By removing this, I'm able to post content with encoded entities.

I've tested this on content which includes MarkDown, HTML, encoded entities, unencoded entities, etc.

### Other information:


## Testing instructions:






* Write a post which includes the literal text `The new &lt;search&gt; element is <em>&quot;gr&euml;át&quot;</em> - It makes me &#x1F970; and &#128573; - you should spend your &pound;s on it.`
* Under the old version, the `<search>` element would be inserted and the rendering of the post would break.
* After this patch, the entities are unchanged and the post displays as 

> The new &lt;search&gt; element is *"grëát"* - It makes me 🥰 and 😽 - you should spend your £s on it.
